ddl/column.go

```diff
@@ -125,7 +125,10 @@ func (d *ddl) onAddColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	// if errorBeforeDecodeArgs {
 	// 	return ver, errors.New("occur an error before decode args")
 	// }
-
+	if err = checkAddColumnTooManyColumns(len(tblInfo.Columns)); err != nil {
```

Comment:
I think `tblInfo.Columns` doesn't the old columns when the new column's state isn't `StateNone`.

Reply:
@zimulala execute the `AddColumn` once and add only one column, so the old columns plus 1 is the new column's state.